docs/04_protocol/02_transactions_protocol.md

```diff
@@ -227,7 +227,7 @@ The process to verify a transaction is twofold. First, the public keys associate
 
 ### 3.4.1. Transaction Context
 
-After the public keys are recovered, a transaction context is created from the transaction instance. This context keeps track of the trace of actions and the action receipt generated as each action is dispatched and executed. All state generated is kept within a transaction trace instance and a list of action receipts. The transaction trace consists of a list of action traces. Each action trace contains information about the executed action, such as the action receipt, the action instance, whether it is a context-free action, and the transaction ID that generated the action. The action receipt is generated later during transaction execution and finalization.
+After the public keys are recovered, a transaction context is created from the transaction instance. This context keeps track of the trace of actions and the action receipt generated as each action is dispatched and executed. All state generated is kept within a transaction trace instance and a list of action receipts. The transaction trace consists of a list of action traces. Each action trace contains information about the executed action, which includes the action receipt, the action instance, whether it is a context-free action, and the transaction ID that generated the action. The action receipt is generated later during transaction execution and finalization.
```

Comment:
"After" seems odd to me as the first word of a paragraph - maybe change to "Once"

Reply:
"This context" - perhaps "The transaction context keeps ..."

Reply:
Yes to both suggestions.